### PR TITLE
Get starkware governors

### PR DESCRIPTION
**Pull request by michalsidzej**
This PR allows getting a full list of starkware proxy governors.

### CODE REVIEW

File: packages/config/src/layer2s/dydx.ts [hunk 1]
```diff
@@ -136,38 +134,22 @@ export const dydx: Layer2 = {
...
-      accounts: [
-        {
-          address: EthereumAddress(
-            '0x8129b737912e17212C8693B781928f5D0303390a',
-          ),
-          type: 'EOA',
-        },
-      ],
+      name: 'Operators',
+      accounts: getProxyGovernance(discovery, 'StarkPerpetual'),
```
Comment (antooni) [inline packages/config/src/layer2s/dydx.ts:152]: fix this

File: packages/shared/src/types/Discovery.ts [hunk 2]
```diff
@@ -85,6 +85,7 @@ export interface StarkWareProxyUpgradeability {
   upgradeDelay: number
   isFinal: boolean
   useConstantDelay?: boolean
+  proxyGovernance?: EthereumAddress[]
```
Comment (antooni) [inline packages/shared/src/types/Discovery.ts:88]: add a comment

**Review by antooni — CHANGES_REQUESTED** [review]
please add requested changes

File: packages/backend/discovery/deversifi/discovered.json [hunk 3]
```diff
@@ -19,23 +19,26 @@
...
         ],
         "getRegisteredVerifiers": [
           "0x6e3AbCE72A3CD5edc05E59283c733Fd4bF8B3baE"
         ],
-        "getSequenceNumber": 3277,
-        "getVaultRoot": "2034467869021769495664402500002349695634078715056798288247388799228725948300",
+        "getSequenceNumber": 3314,
+        "getVaultRoot": "2457293978998501030141425210957102520864271926575026367356632712756675268614",
         "getVaultTreeHeight": 31,
         "GOVERNORS": [
```
Comment (antooni) [inline packages/backend/discovery/deversifi/discovered.json:43]: remove this from discovery json

File: packages/backend/discovery/sorare/discovered.json [hunk 4]
```diff
@@ -11,7 +11,11 @@
...
         "upgradeDelay": 259200,
-        "isFinal": false
+        "isFinal": false,
+        "proxyGovernance": [
+          "0x5918481F777dBe437De249492B90AffB4e655de4",
+          "0xCc928977e4a75d25099e7DA7B6Fd79Dac2f9fD2B"
+        ]
       },
       "values": {
         "getUpgradeActivationDelay": 259200,
```
Comment (antooni) [inline packages/backend/discovery/sorare/discovered.json:21]: remove GOVERNORS

**Review by antooni — CHANGES_REQUESTED** [review]
remove unused GOVERNORS field

File: packages/backend/discovery/immutablex/discovered.json [hunk 5]
```diff
@@ -11,7 +11,10 @@
...
         "implementation": "0xB8563AD5aF1F79dd04937BE8B572318c8e6f43AC",
         "upgradeDelay": 1209600,
-        "isFinal": false
+        "isFinal": false,
+        "proxyGovernance": [
+          "0x799Ec7ff0Bf9d8Bb4f3643aa85dA0971b1104B5a"
+        ]
       },
       "values": {
         "getUpgradeActivationDelay": 1209600,
```
[the remark this answers is not in the document]
Comment (michalsidzej) [reply packages/backend/discovery/immutablex/discovered.json:20]: I will create a separate PR adding discovery to immutablex .ts file

**Review by antooni — APPROVED** [review]

**Pull request merged**